packages/jest-util/src/__tests__/create_process_object.test.js

```diff
@@ -5,14 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import EventEmitter from 'events';
 import createProcessObject from '../create_process_object';
 
 it('creates a process object that looks like the original one', () => {
   const fakeProcess = createProcessObject();
 
   // "process" inherits from EventEmitter through the prototype chain.
-  expect(fakeProcess instanceof EventEmitter).toBe(true);
+  expect(typeof fakeProcess.on).toBe('function');
```

Comment:
So `fakeProcess instanceof EventEmitter` returns false now? Maybe worth to update the comment.

Reply:
This is unfortunate, but yes. Because `EventEmitter` is re-evaluated now, it is not the parent `EventEmitter` anymore. In order to get that check right we should build the process object _inside_ the VM context. However, this requires being able to do `require('events')` which overcomplicates all the current flow.

The comment is still valid, since we check the existence of the `on` & various methods; but I can clarify it.